src/main/java/seedu/address/model/order/Order.java

```diff
@@ -58,8 +58,7 @@ public boolean isSameOrder(Order otherOrder) {
             return true;
         }
 
-        return otherOrder != null
-                && otherOrder.getPhone().equals(getPhone());
+        return otherOrder != null;
```

Comment:
This means:
```
if otherOrder is null:
    return False
else:
    return true
```

Wont this mean that as long as the `otherOrder` is not null, `isSameOrder` will be True, which it shouldnt be?

Reply:
Also, might want to change the JavaDoc comment for this

Reply:
I discussed this with Gerald; This method was originally for consistency with Person class, but since there is no need to check if an Order is a duplicate, then overriding the equals method is sufficient. Thanks for highlighting it